### ISSUE TITLE
showErrorsOnChange => Only after first error

### ISSUE THREAD

**Issue by jonasdumas** [foxhound87/mobx-ajv-form#354]
Hello, Thank you for your great work on that project.

It would be nice to have another option feature : showErrorsOnChangeAfterFirstBlur

It's quite frustrating to get an error when we're currently filling a field and it's not finish. It's probably why showErrorsOnChange is defined to false by default in Mobx-react-form.
But then, when we've an error in a field because of a mistake ... if we correct it, it doesn't show if it's fine before onBlur. It would be nice to have a new behavior which showErrorsOnChange on a field only if there was already an error set on it. Something like: 
showErrorsOnChangeAfterFirstBlur
showErrorsOnChangeAfterAnError

**Comment by foxhound87** [comment]
@jonasdumas Thanks for the suggestion. I will think about it.

**Comment by oleghein** [comment]
@foxhound87 thanks for the awesome work!

I was searching for this option as well. It would be a huge UX benefit.

**Comment by foxhound87** [comment]
Introduced by #477
